### PR TITLE
Clear marketplace cache during upgrade

### PR DESCRIPTION
**Pull request by J-N-K**
Related to https://github.com/openhab/openhab-core/pull/4019

### CODE REVIEW

File: resources/control-runtime/preinst [hunk 1]
```diff
@@ -14,13 +14,15 @@ set -e
...
 UD_TMP=/var/lib/openhab/tmp
+UD_MARKETPLACE=/var/lib/openhab/marketplace
 UD_KAR=/var/lib/openhab/kar
 RESTART_FLAG_FILE=/var/lib/openhab/.restartOnUpgrade
 OLD_VERSION_FILE=/var/lib/openhab/.previousVersion
 
 removeCache(){
   [ -d ${UD_CACHE} ] && rm -rf ${UD_CACHE}
   [ -d ${UD_TMP} ] && rm -rf ${UD_TMP}
+  [ -d ${UD_MARKETPLACE} ] && rm -rf ${UD_TMP}
```
Comment (BClark09) [inline resources/control-runtime/preinst:25]:
```suggestion
  [ -d ${UD_MARKETPLACE} ] && rm -rf ${UD_MARKETPLACE}
```

**Review by BClark09 — CHANGES_REQUESTED** [review]
Thanks @J-N-K! With the suggested change should we wait until your PR in core is merged?

**Comment by J-N-K** [comment]
Correct, this will otherwise create issues.

**Review by BClark09 — APPROVED** [review]
Great, I'll watch the other PR and merge when appropriate. Thanks again!

**Comment by J-N-K** [comment]
@BClark09 You can merge now

**Pull request merged**